src/DssSpell.t.sol

```diff
@@ -891,129 +864,31 @@ contract DssSpellTest is DssSpellTestBase {
 
     // SPELL-SPECIFIC TESTS GO BELOW
 
-    function testDirectSparkMorphoIntegration() public {
-        bytes32 ilk         = "DIRECT-SPARK-MORPHO-DAI";
-        address vault       = 0x73e65DBD630f90604062f6E02fAb9138e713edD9;
-        address operator    = 0x298b375f24CeDb45e936D7e21d6Eb05e344adFb5;
-        uint256 debtCeiling = 100 * MILLION * WAD;
+    function testLineMomAddedIlks() public {
```

Comment:
Can we also test `wipe` here to ensure MOM would function correctly on emergency case?

Reply:
Make sense, we could check if the value gets updated properly but I don't think testing implementation of MOM contract functionality is scope of the spell. Thanks for the input :)